src/components/molecules/OcTable/OcTable.vue

```diff
@@ -248,8 +255,34 @@ export default {
     },
   },
   methods: {
-    dragStart(file) {
+    dragOver(event) {
+      event.preventDefault()
+      event.target.style.cursor = "move"
+    },
+    setGhostElement(file, event) {
+      const selection = this.selection
```

Comment:
```suggestion
      const selection = [...this.selection]
```
so that we can work on a shallow copy of the array